### PR TITLE
US#35: Estimate effectiveness

### PR DESCRIPTION
**Pull request by ShashankReddyBand**
Add a new chart for estimated effectiveness.

### CODE REVIEW

File: client/actions/project.ts [hunk 1]
```diff
@@ -137,3 +137,24 @@ export async function getWorkCapacity(projectSlug: string) {
...
 }
+
+export async function getEstimateEffectiveness(milestoneId: string) {
+    const Response = z.array(z.object({
+        storyTitle: z.string(),
+        effectiveness: z.number()
+    }))
+    
+
+    const url = `http://localhost:8080/api/${milestoneId}/getEstimateEffectiveness`;// need to change the url
```
Comment (AshankDsouza) [inline client/actions/project.ts:148]: please avoid hardcoding. this will now become a bug since we have a deployed site

Comment (AshankDsouza) [inline client/actions/project.ts]: please avoid hardcoding. this will now become a bug since we have a deployed site

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "interactive"
```
Comment (AshankDsouza) [inline .vscode/settings.json]: please delete this file

**Review by AshankDsouza — CHANGES_REQUESTED** [review]
Please could we resolve these comments?

**Review by ShashankReddyBand — COMMENTED** [review]
Addressed the comments

**Review by AshankDsouza — APPROVED** [review]

**Pull request merged**